app/styles/components/_loader.scss

```diff
@@ -19,30 +19,71 @@
 }
 
 .animate-rotate {
-	animation: animate-rotate 400ms linear infinite;
+	animation: animate-rotate 1600ms linear infinite;
```

Comment:
much better

Reply:
yes, the css anbimation above can be worked on, it was jst a test to make a 1210 turntable, and maybe other animations on different pages. just a test